students/slupek_bartosz/lesson_00_the_setup/hello_world.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+
+"""This is an example module to teach you about git and PyCharm use.
+
+Python supports many ways to write programs. You can start with a very simple
+imperative code, like this script.
+"""
+
+#!/usr/bin/env python3
```

Comment:
To jest tak zwane 'shebang line', która daje możliwość uruchomienia skryptu z wiersza poleceń bez konieczności użycia 'python'. Linia powinna znajdować się na początku pliku i tylko raz, tak więc jej wystąpienie jest zbędne.
W drugim kommicie powinienes zastąpić zawartośc tego skryptu zawartością z importable_module.py